### DISCUSSION TITLE
Is it possible to update seasons?  Like say, on a daily basis, by re-importing updated tables?

### DISCUSSION THREAD

**Discussion by IndigoAK** [tbrittain/SmbExplorerCompanion#196] [General]
I'm receiving a vague error trying to update a season (by exporting new tables and importing them into Explorer with the current season selected) and I wanted to check if this was even something that app was intended for before I submitted a bug report.

**Comment by tbrittain** [comment]
Yes in theory it should work. The way that it is configured is that upon re-importing an existing season, it should just replace the batting/pitching stats for players and schedule/stats for teams as well

**Comment by IndigoAK** [comment]
I guess I will submit a bug report then.  I am trying to update a league in progress using this method and getting an error that says it's failing to save the entity changes and to see the inline error.  It properly imports all the teams, but fails on the first player in the table.

**Reply by tbrittain** [reply]
Darn yup then something is going wrong with it. Definitely another bug that I want to resolve because that should be supported. Apologies again
